app/server/models/transfer_account.py

```diff
@@ -86,17 +95,17 @@ def get_approval(self, receiving_address):
     @hybrid_property
     def total_sent(self):
         return int(
-            db.session.query(func.sum(CreditTransfer.transfer_amount).label('total')).execution_options(show_all=True)
-            .filter(CreditTransfer.transfer_status == TransferStatusEnum.COMPLETE)
-            .filter(CreditTransfer.sender_transfer_account_id == self.id).first().total or 0
+            db.session.query(func.sum(server.models.credit_transfer.CreditTransfer.transfer_amount).label('total')).execution_options(show_all=True)
```

Comment:
are we organizing our code incorrectly? this circular dependency business feels like we're doing something wrong...

Reply:
Hmmm yeah doesn't feel right  does it?
I tried the accepted answer here:
https://stackoverflow.com/questions/29217076/flask-sqlalchemy-query-have-to-import-all-models/29218728

But couldn't even get the imports in `__init.py__` to recognise